wordpress-mobile/WordPress-Aztec-iOS
# Refactor HTML related enums to single type outside of element node.

This PR refactors code related to HTML elements, consolidating all the html element related code in a single enum.

How to test:

 - Build the app
 - Run the tests
 - Run the demo app and see if behaviour is still the same.
